.github/scripts/docker-create.sh

```diff
@@ -298,6 +298,17 @@ generate_test_data() {
   fi
 }
 
+download_dot_net_binaries() {
+  echo "downloading dotnet binaries"
+  rm ../../src/main/resources/executables/wrongsecrets-dotne*
+  curl -L -o ../../src/main/resources/executables/wrongsecrets-dotnet https://github.com/OWASP/wrongsecrets-binaries/releases/download/0.1.0/wrongsecrets-dotnet
```

Comment:
```suggestion
  curl -L -o ../../src/main/resources/executables/wrongsecrets-dotnet https://github.com/OWASP/wrongsecrets-binaries/releases/download/$BINARY_VERSION/wrongsecrets-dotnet
```

Reply:
Where do you want to maintain the variable? Till now I am not sure if we will have a another release soon ;-)

Reply:
Feel free to add,can you set the version on the top of the script as well please :-)?